dts/bindings/i3c/snps,designware-i3c.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020 Samsung Electronics Co., Ltd.
+# Copyright (C) 2023 Meta Platforms
+# SPDX-License-Identifier: Apache-2.0
+
+description: >
+    This is a binding for the Synopsys I3C interface
+
+compatible: "snps,designware-i3c"
+
+include: i3c-controller.yaml
+
+properties:
+  input-clock-frequency:
+    required: true
+    type: int
+    description: clk frequency.
+
+  od-thigh-max-ns:
+    type: int
+    default: 41
```

Comment:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
This comes from the I3C Specification.... but I can add a note there

Reply:
yes, usually defaults do have some justification behind them, that's why we need to put it in the description